src/MusicStore/Platform.cs

```diff
@@ -82,5 +82,14 @@ public bool IsRunningOnNanoServer
                 return _isNano.Value;
             }
         }
+
+        //Sql client not available on mono, non-windows, or nano
+        public bool UseInMemoryStore
```

Comment:
I think it's reasonable for the platform class to provide indication whether SQL is available. Maybe I should rename this?